src/strings.jl

```diff
@@ -19,6 +19,7 @@ const esriid2code = Dict(
   "North_Pole_Orthographic" => ESRI{102035},
   "South_Pole_Orthographic" => ESRI{102037},
   "TM75_Irish_Grid" => EPSG{29903},
+  "ETRS_1989_UTM_Zone_32N" => EPSG{25832},
```

Comment:
Please add this entry in alphabetical order as explained in the comment above.

Reply:
Sorry, I originally used the wrong Esri ID. Now it should be correct.